scss/standalone/patterns_accordion.scss

```
@@ -1,4 +1,5 @@
 @import '../vanilla';
 @include vf-base;
 
+@include vf-p-form-tick-elements;
```

Is this correct? it does seem to render correctly now https://vanilla-framework-3963.demos.haus/docs/examples/patterns/accordion/tick-elements

---

Yes, perfect, I'd just add a comment explaining why it is needed, like so:

```suggestion
// needed for the example with checkboxes
@include vf-p-form-tick-elements;
```